pkg/workloads/cortex/tf_api/api.py

```diff
@@ -16,6 +16,7 @@
 import os
 import argparse
 import time
+import inspect
```

Comment:
nit: `inspect` is not used I think, can you check some of the other ones as well?